packages/server/examples/gliders.html

```diff
@@ -0,0 +1,206 @@
+<m-group>
```

Comment:
Looks like this wasn't meant to be left in?

Reply:
Removed in [commit 6](https://github.com/mml-io/mml-playground/pull/21/commits/20f1bded3737d13a629d5d7aea2fdf44b2c4cd4f)